Week8/app/src/main/java/com/em4n0101/mytvshows/viewModels/PersonsViewModel.kt

```diff
@@ -0,0 +1,26 @@
+package com.em4n0101.mytvshows.viewModels
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.asLiveData
+import androidx.lifecycle.viewModelScope
+import com.em4n0101.mytvshows.model.Person
+import com.em4n0101.mytvshows.repositories.PersonsRepository
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+class PersonsViewModel(application: Application) : AndroidViewModel(application) {
```

Comment:
I suggest you pass in the repository as a constructor parameter and to use `ViewModel` instead of `AndroidViewModel`